packages/get-platform/src/__tests__/parseOpenSSLVersion.test.ts

```diff
@@ -1,28 +1,55 @@
-import { parseOpenSSLVersion } from '../getPlatform'
-
-const tests = [
-  {
-    name: '1.1',
-    content: `OpenSSL 1.1.0d  1 Feb 2014`,
-    expect: '1.1.x',
-  },
-  {
-    name: '1.2',
-    content: `OpenSSL 1.0.2g  1 Mar 2016`,
-    expect: '1.0.x',
-  },
-  {
-    name: '3.0',
-    content: `OpenSSL 3.0.2 15 Mar 2022 (Library: OpenSSL 3.0.2 15 Mar 2022)`,
-    expect: '3.0.x',
-  },
-]
+import { parseLibSSLVersion, parseOpenSSLVersion } from '../getPlatform'
 
 describe('parseOpenSSLVersion', () => {
-  for (const t of tests) {
+  const tests = [
+    {
+      name: 'openssl 1.1',
+      content: `OpenSSL 1.1.0d  1 Feb 2014`,
+      expect: '1.1.x',
+    },
+    {
+      name: 'openssl 1.0.2',
+      content: `OpenSSL 1.0.2g  1 Mar 2016`,
+      expect: '1.0.x',
+    },
+    {
+      name: 'openssl 3.0',
+      content: `OpenSSL 3.0.2 15 Mar 2022 (Library: OpenSSL 3.0.2 15 Mar 2022)`,
+      expect: '3.0.x',
+    },
+  ]
+
+  describe.each(tests)('$name', (t) => {
     test(t.name, () => {
       const actual = parseOpenSSLVersion(t.content)
       expect(actual).toBe(t.expect)
     })
-  }
+  })
+})
+
+describe('parseLibSSLVersion', () => {
+  const tests = [
+    {
+      name: 'libssl 1.0',
+      content: `/lib/libssl.so.1`,
+      expect: '1.0.x',
+    },
+    {
+      name: 'libssl 1.1',
+      content: `/lib/libssl.so.1.1`,
+      expect: '1.1.x',
+    },
+    {
+      name: 'libssl 3.0',
+      content: `/lib/libssl.so.3`,
+      expect: '3.0.x',
+    },
```

Comment:
Maybe also add test cases for the full version (just three numbers and also three numbers + letter). For example rhel has both `/lib64/libssl.so.1.1` (as a symlink) and `/lib64/libssl.so.1.1.1g` (as the real file), would be nice to know that we can parse both.